### DISCUSSION TITLE
What do you think about Linux Mint (19.3 64 bits) as LTSP server?

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#228] [Q&A]
Hello! 
I am trying to config Linux Mint LTSP for office (small office). All steps to install LTSP ok. When client up, appear 2 erros:

1 - uncompress error...and stop boot or
2 - after ipxe check - 3 lines count percent It stops and not run more.

I will do more tests other NIC or clients 

So, can be a distro problem or is better other (ubuntu mate....)

Thank you

**Comment by alkisg** [comment]
Can you upload a photo of the client message?

Is it possible to be a network issue, e.g. bad cabling?

Can you try with another client that has a different network card?

**Comment by doguibnu** [comment]
> Can you upload a photo of the client message?

Yes!
 
> Is it possible to be a network issue, e.g. bad cabling?

I will go check cables

> Can you try with another client that has a different network card?

Yes @alkisg I will

Follow the pics

![00-IMG_20200110_081704](https://user-images.githubusercontent.com/38897311/72151812-fdcea000-3387-11ea-9a7c-eba232ee74d9.jpg)
![02-IMG_20200110_081719](https://user-images.githubusercontent.com/38897311/72151813-0030fa00-3388-11ea-900e-88c900949ea8.jpg)
![03-IMG_20200110_081852](https://user-images.githubusercontent.com/38897311/72151816-01fabd80-3388-11ea-88ff-dcbde2a4db8c.jpg)
![04-IMG_20200110_082118 (cópia)](https://user-images.githubusercontent.com/38897311/72151819-032bea80-3388-11ea-81f0-b1cb0f69543f.jpg)
![05-IMG_20200110_081925](https://user-images.githubusercontent.com/38897311/72151821-045d1780-3388-11ea-9d21-28ca5e9c129e.jpg)
![06-IMG_20200110_081943](https://user-images.githubusercontent.com/38897311/72151827-07580800-3388-11ea-8bdf-7988f35791e2.jpg)

**Comment by doguibnu** [comment]
see the pics can be bad cable to client side!

**Comment by doguibnu** [comment]
Well!

I get other client, it ups but not accept login and password from users. That crazy!
![loginIMG_20200110_105231](https://user-images.githubusercontent.com/38897311/72158188-69206e00-3398-11ea-8f50-3a50ba081352.jpg)

**Comment by alkisg** [comment]
As seen via IRC/VNC, the main problem was missing packages (at least sshfs and epopes-client).

All Mint users should remember to use `--install-recommends` when installing packages. Mint breaks Debian policy by using --no-install-recommends by default.
